lib/classification_lifecycle.rb

```diff
@@ -77,7 +77,9 @@ def update_seen_subjects
   def publish_data
     return unless classification.complete?
 
-    PublishClassificationWorker.perform_async(classification.id)
+    unless project.disabled_data_export?
```

Comment:
What do data exports have to do with the internal service bus we run?

Reply:
the method can be renamed to reflect pushing data to the stream as well as csv dumps. Both are exports in my mind just different mechanisms.